pywemo/ouimeaux_device/insight.py

```diff
@@ -80,6 +81,11 @@ def today_kwh(self):
     def current_power(self):
         """Return the current power usage in mW."""
         return self.insight_params['currentpower']
+    
```

Comment:
It looks like there's whitespace on this line, and that's why the CI tests failed. Can you please remove the whitespace?

Reply:
Ok i remove the whitespace